hat/assets/js/apps/Iaso/domains/registry/components/OrgUnitInfos.tsx

```diff
@@ -65,18 +65,6 @@ export const OrgUnitInfos: FunctionComponent<Props> = ({
                             {!orgUnitType && orgUnit.org_unit_type_name}
                         </TableCell>
                     </TableRow>
-                    <TableRow>
-                        <TableCell className={classes.leftCell}>
-                            {formatMessage(MESSAGES.status)}
-                        </TableCell>
-                        <TableCell>
-                            {MESSAGES[orgUnit.validation_status]
-                                ? formatMessage(
-                                      MESSAGES[orgUnit.validation_status],
-                                  )
-                                : '--'}
-                        </TableCell>
-                    </TableRow>
```

Comment:
I don't understand why remove that from registry? Is that part of another ticket?

Reply:
same question

Reply:
Because the org unit tree ill always ask for validated org units, so the info is not relevant